danny-avila/LibreChat
# 🪲 fix(useTextarea): enterToSend bugs

## Summary
This PR introduces changes to the `handleKeyDown` function in `useTextarea.ts` to improve the behavior of the text input when `enterToSend` is set to false. Now, when `enterToSend` is false, pressing `SHIFT + ENTER` will only create one new line instead of two, and pressing `CTRL + ENTER` will send the message

#2962 

## Change Type

## Testing
To test these changes, follow these steps:
1. Set `enterToSend` to false
2. In the text input, press `SHIFT + ENTER`. It should only create one new line
3. Press `CTRL + ENTER`. It should send the message

## Test Configuration:

### Firefox
- OS: Windows 11
- Browser: Firefox
- Version: 127.0b9 (64-bit)

### Chrome
- OS: Windows 11
- Browser: Chrome
- Version: 125.0.6422.142 (64-bit)

## Checklist
